docs/docs/02-displaying-data.it-IT.md

```diff
@@ -21,7 +21,7 @@ Diamo un'occhiata ad un esempio davvero semplice. Creiamo un file dal nome `hell
     <title>Hello React</title>
     <script src="https://unpkg.com/react@{{site.react_version}}/dist/react.js"></script>
     <script src="https://unpkg.com/react-dom@{{site.react_version}}/dist/react-dom.js"></script>
-    <script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
```

Comment:
Babel-standalone is on CDNJS and I've seen better perf with CDNJS compared to unpkg (also it has more users so it's more likely to hit the cache). Maybe use CDNJS instead?